src/ArborX_LinearBVH.hpp

```diff
@@ -192,7 +230,7 @@ BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
 
   if (size() == 1)
   {
-    Kokkos::View<size_t *, DeviceType> permutation_indices("permute", 1);
+    Kokkos::View<size_t *, MemorySpace> permutation_indices("permute", 1);
```

Comment:
I'm not sure what's happening here. If you provide `MemorySpace` for a view constructor, and initialize it, it will use the default execution space for it, which could be different from the `ExecutionSpace` you passed as argument (just the type, even disregarding instance). Would not it be more valid to define `DeviceType<ExecutionSpace,MemorySpace>` and use it for views? I think I forgot this point in other PRs.

Also, is there a plan in Kokkos to modify `View` to support constructors taking in execution space?

Reply:
Good catch, I need to pass `space` here as well.